ElectronClient/app.js

```diff
@@ -1025,6 +1025,13 @@ class Application extends BaseApplication {
 					click: () => {
 						Setting.incValue('windowContentZoomFactor', 10);
 					},
+					accelerator: 'CommandOrControl+Plus',
+				}, {
+					label: _('Zoom In'),
```

Comment:
Since the item is not visible, can we remove the label? Or does it not working if there's no label?

Reply:
No, when we remove the label, it won't work. But we can set it to an empty string.
But I think we should leave it as is.

Reply:
@laurent22 not aure, if you saw my comment.

Reply:
I think this is fine, but please could you add a comment above to explain why the menu item is duplicated?

Reply:
I will do that later this evening. Sorry, I didn't see your comment earlier. I actually wanted to do that anyway, but forgot. Thanks for the reminder.

Reply:
done

Reply:
Is the comment ok?